### DISCUSSION TITLE
Question Add Mods

### DISCUSSION THREAD

**Discussion by spyzzz** [wolveix/satisfactory-server#14] [Q&A] [answered]
Hello 

First, i would like to thank you for your work. 

I was able to run the server in less than 10min using docker on synology.

all is working fine, but i didnt succeed on installing mods on the server.

I tried to unzip SML.zip in /config/gamefiles/FactoryGame/Mods/

And then i putted some mods in (like permaday.smod) in

/config/gamefiles/FactoryGame/Mods/permaday.smod

I restarted the docker, but the mod is not working

Thanks for help

**Comment by wolveix** [comment, marked as answer]
Hello! This is a duplicate of #8. Mods are not officially supported as I have not tested them. Adding support should be simple enough, I just haven't had the time to implement it myself due to moving home. PRs are always welcome! Please continue this discussion over at #8 rather than here :)
